### ISSUE TITLE
Slash command "/simc" no longer works? (Possible conflict with ElvUI?)

### ISSUE THREAD

**Issue by CursedShadow** [simulationcraft/simc-addon#10]
When attempting to use the slash command, it does nothing and does not get entered into the chat. Instead it throws the following error:

```
12x Simulationcraft\core.lua:88: attempt to compare number with table
Simulationcraft\core.lua:88: in function <Simulationcraft\core.lua:85>
Simulationcraft\core.lua:119: in function <Simulationcraft\core.lua:100>
Simulationcraft\core.lua:611: in function `PrintSimcProfile'
Simulationcraft\core.lua:65: in function `?'
...ns\ElvUI\Libraries\AceConsole-3.0\AceConsole-3.0-7.lua:94: in function `?'
FrameXML\ChatFrame.lua:4547: in function `ChatEdit_ParseText'
FrameXML\ChatFrame.lua:4215: in function `ChatEdit_SendText'
FrameXML\ChatFrame.lua:4251: in function <FrameXML\ChatFrame.lua:4247>
[C]: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>
[C]: ?

Locals:
char = TestButton {
 0 = <userdata>
 Text = TestButtonText {
 }
 Right = <unnamed> {
 }
 Middle = <unnamed> {
 }
 Left = <unnamed> {
 }
}
(*temporary) = nil
(*temporary) = "attempt to compare number with table"
```

**Comment by CursedShadow** [comment]
UPDATE: After disabling ElvUI, the error persists.

**Comment by seriallos** [comment]
Releasing a fix now, 1.9.1 should be available shortly (once Curse picks up the changes)
